CHARTER.md

```diff
@@ -0,0 +1,138 @@
+
+### Charter and Governing Documents
+
+The Working Group may, from time to time, adopt or amend its Governing Documents and Charter, using a modified Call for Agreement process:
+* A quorum of at least 51% of active Working Group Members must vote.
+* A quorum of 3 Chairs is needed
+* Two-thirds of the voting quorum must approve the proposal.
+* A majority of Chairs must approve the proposal.
+* A public notice period of no less than 14 days from the Call for Agreement proposing the change must elapse before voting begins.
+* Any Call for Agreement that follows this process is considered a Governing Document. 
+
+For initial approval of this Charter via Call for Agreement all members are eligible to vote, even those that have been a member for less than 21 days. This Charter will be approved if there is a majority of positive votes.
+
+## Organizational Roles
+### Role of Chairs
+The primary role of Chairs is to run operations and the governance of the group. The Chairs are responsible for:
+* Setting the agenda for meetings.
+* Extending discussion via asynchronous communication to be inclusive of members who cannot attend a specific meeting time.
+* Scheduling discussing of proposals that have been submitted.
+* Asking for new proposals to be made to address an identified need.
+* Oversee disciplinary action for members. The Chairs have the authority to declare a member inactive and expel members for cause.  
+* Chairs will serve for one-year revolving terms and will be approved using the [Condorcet Method](https://en.wikipedia.org/wiki/Condorcet_method). Upon the expiration of a Chair’s term, it will continue for another year, unless challenged. 
+
+The terms for founding Chairs start on the approval of this charter. 
+
+When no candidate has submitted their name for consideration, the current Chairs may appoint an acting Chair until a candidate comes forward.
+
+Chairs must be active members. Any inactivity, disability or ineligibility results in immediate removal.
+
+Chairs may be removed by petition to the Working Group through the Call for Agreement process outlined above.
+
+Additional Chairs may be added so long as the existing number of Chairs is odd. These Chairs are added using a Call for Agreement. Extra Chairs enjoy the same rights, responsibilities, and obligations of a Chartered Chair. Upon vacancy of an Extra Chair, it may be filled by appointment by the remaining Chairs, or a majority vote of the Working Group until the term naturally expires.
+
+In the event that an even number of Chairs exist and vote situation arises, the Chairs will randomly select one chair to abstain. 
+
+### Role of Sub-Working Groups 
+Each Sub-Working Group (SWG) must have a Chair working as an active sponsor. Under the mandate of the Working Group, each SWG will have the autonomy to establish their own charter, membership rules, meeting times, make in-scope decisions as delegated, and management processes.
```

Comment:
Grammar did not make sense.  "establish...make in-scope decisions".  Moved it to its own sentence.

```suggestion
Each Sub-Working Group (SWG) must have a Chair working as an active sponsor. Under the mandate of the Working Group, each SWG will have the autonomy to establish their own charter, membership rules, meeting times, and management processes. Each SWG will also have the authority to make in-scope decisions as delegated by the Working Group.
```